swig/cpp/src/Sysrepo.h

```diff
@@ -0,0 +1,66 @@
+#ifndef Sysrepo_H
+#define Sysrepo_H
+
+#include <iostream>
+
+extern "C" {
+#include "sysrepo.h"
+}
+
+class Error
+{
+public:
+    size_t cnt;
+    const sr_error_info_t *info;
+    void log_stderr(sr_log_level_t log_level);
```

Comment:
I think this could be the class method, the Error instance is not needed to alter the log level